src/pages/api/v1/applications/index.ts

```diff
@@ -0,0 +1,46 @@
+import prisma from '../../../../lib/prisma';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+const getApplications = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const data = await prisma.application.findMany();
+    if (data) {
+      res.status(200).json(data);
+    } else {
+      res
+        .status(404)
+        .json({ message: `Could not find Applications try again` });
+    }
+  } catch (error) {
+    res.status(500).json({
+      message:
+        'Something went wrong from our side and we were unable to get Applications.',
+    });
+  }
+};
+
+const postHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  try {
+    const { jobListingId, status } = req.body;
+    const data = await prisma.application.create({
+      data: { jobListingId, status },
+    });
+    if (data) {
```

Comment:
I think we need to consider what happens when data is a falsy value.

(I'm not sure myself. I think we should look into this one together)